### DISCUSSION TITLE
Add permission for chorus teleport

### DISCUSSION THREAD

**Discussion by DextralityNetwork** [BG-Software-LLC/SuperiorSkyblock2#2023] [Approved Feature Requests]
Several servers we have seen the possibility of adding this permission because certain users tend to want to deny this use of teleportation, whether chorus or enderpearl, so that other people can leave the zone of the stores that they assign.

**Comment by Lemvs** [comment]
@DextralityNetwork This has been added as a CHORUS_FRUIT permission for some time.
@OmerBenGera you can close it.
